<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CExternalTools</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CExternalTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="NSABUtils::CPathBasedDelayLineEdit" name="mkvPropEditExe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>mkvmerge Executable:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Download from here &lt;a href=&quot;https://www.matroska.org/downloads/mkvalidator.html&quot;&gt;https://www.matroska.org/downloads/mkvalidator.html&lt;/a&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="NSABUtils::CPathBasedDelayLineEdit" name="ffprobeExe"/>
   </item>
   <item row="6" column="2">
    <widget class="QToolButton" name="btnSelectFFProbeExe">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../SABUtils/resources/SABUtils.qrc">
       <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="btnSelectMKVPropEditExe">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../SABUtils/resources/SABUtils.qrc">
       <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="btnSelectMKVValidatorExe">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../SABUtils/resources/SABUtils.qrc">
       <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="NSABUtils::CPathBasedDelayLineEdit" name="ffmpegExe"/>
   </item>
   <item row="5" column="2">
    <widget class="QToolButton" name="btnSelectFFMpegExe">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../SABUtils/resources/SABUtils.qrc">
       <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>mkvpropedit Executable:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>mkvalidator Executable:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>ffmpeg Executable:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnSelectMKVMergeExe">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../SABUtils/resources/SABUtils.qrc">
       <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Download from here &lt;a href=&quot;https://mkvtoolnix.download/downloads.html#windows&quot;&gt;https://mkvtoolnix.download/downloads.html&lt;/a&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ffmpeg and ffprobe need to come from the Emby distribution for BIF creation. &lt;br/&gt;Download from here &lt;a href=&quot;https://emby.media/windows-server.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://emby.media/windows-server.htm&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="NSABUtils::CPathBasedDelayLineEdit" name="mkvValidatorExe"/>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>ffprobe Executable:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="NSABUtils::CPathBasedDelayLineEdit" name="mkvMergeExe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NSABUtils::CPathBasedDelayLineEdit</class>
   <extends>QLineEdit</extends>
   <header>SABUtils/DelayLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mkvMergeExe</tabstop>
  <tabstop>btnSelectMKVMergeExe</tabstop>
  <tabstop>mkvPropEditExe</tabstop>
  <tabstop>btnSelectMKVPropEditExe</tabstop>
  <tabstop>mkvValidatorExe</tabstop>
  <tabstop>btnSelectMKVValidatorExe</tabstop>
  <tabstop>ffmpegExe</tabstop>
  <tabstop>btnSelectFFMpegExe</tabstop>
  <tabstop>ffprobeExe</tabstop>
  <tabstop>btnSelectFFProbeExe</tabstop>
 </tabstops>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
